packages/styles/skip-link.css

```
@@ -1,19 +1,17 @@
 :root {
-  --skip-link-background-color: rgba(11, 14, 17, 0.9);
+  --skip-link-background-color: rgba(255, 255, 255, 0.95);
+  --skip-link-focus-color: var(--focus-light);
+  --skip-link-text-color: var(--text-color-base);
 }
 
 .SkipLink {
-  min-width: 173px;
-  height: 58px;
-  background-color: var(--skip-link-background-color);
   position: absolute;
   box-sizing: border-box;
-  padding: var(--space-half);
+  padding: 8px;
```

We use [variables](https://github.com/dequelabs/cauldron/blob/develop/packages/styles/variables.css) for a majority of our padding in cauldron, this should probably be the following:

```suggestion
  padding: var(--space-smallest);
```